src/libstore/legacy-ssh-store.cc

```diff
@@ -15,6 +15,15 @@
 
 namespace nix {
 
+LegacySSHStoreConfig::LegacySSHStoreConfig(
+    std::string_view scheme,
+    std::string_view host,
```

Comment:
should you stick with "authority" and just add a comment describing what the authority means here?
That way all the APIs can be consistently named.

Reply:
Sure, good idea. https://github.com/NixOS/nix/pull/3425 will really clear this up :)